src/admin-views/aggregator/status.php

```diff
@@ -0,0 +1,210 @@
+		</tr>
+	</thead>
+	<tbody>
+		<?php
+		// Facebook status section
+		$indicator = 'good';
+		$notes = '&nbsp;';
+		$text = '&nbsp;';
+		if ( Tribe__Events__Aggregator::instance()->api( 'origins' )->is_oauth_enabled( 'facebook' ) ) {
+			if ( ! Tribe__Events__Aggregator__Settings::instance()->is_fb_credentials_valid() ) {
+				$indicator = 'warning';
+				$text = __( 'You have not connected Event Aggregator to Facebook', 'the-events-calendar' );
+				$facebook_auth_url = Tribe__Events__Aggregator__Record__Facebook::get_auth_url( array( 'back' => 'settings' ) );
+				$notes = '<a href="' . esc_url( $facebook_auth_url ). '">' . _x( 'Connect to Facebook', 'link for connecting facebook', 'the-events-calendar' ) . '</a>';
+			}
+		} else {
+			$indicator = 'warning';
+			$text = __( 'Facebook oAuth is currently disabled', 'the-events-calendar' );
+			$notes = __( 'Some types of Facebook events may not be importing correctly (ex: private, group, 18+)', 'the-events-calendar' );
+		}
+		?>
+		<tr>
+			<td class="label">
+				<img src="<?php echo tribe_events_resource_url( 'images/aggregator/facebook.png' ); ?>" />
+				<span><?php esc_html_e( 'Facebook', 'the-events-calendar' ); ?></span>
+			</td>
+			<td class="indicator <?php esc_attr_e( $indicator ); ?>"><span class="dashicons dashicons-<?php echo esc_attr( $indicator_icons[ $indicator ] ); ?>"></span></td>
+			<td><?php echo esc_html( $text ); ?></td>
+			<td><?php echo $notes; ?></td>
+		</tr>
+		<?php
+		// Meetup status section
+		$indicator = 'good';
+		$notes = '&nbsp;';
+		$text = '&nbsp;';
+		$meetup_api_key = tribe_get_option( 'meetup_api_key' );
+		if ( ! $meetup_api_key ) {
+			$indicator = 'warning';
+			$text = __( 'You have not set your Meetup API key.', 'the-events-calendar' );
+			$notes = sprintf( // add link to API tab
+				__( 'Add your API key on the %1$sSettings &gt; APIs%2$s page', 'the-events-calendar' ),
+				'<a href="' . esc_url( Tribe__Settings::instance()->get_url( array( 'tab' => 'addons' ) ) ) . '">',
+				'</a>'
+			);
+		}
+		?>
+		<tr>
+			<td class="label">
+				<img src="<?php echo tribe_events_resource_url( 'images/aggregator/meetup.png' ); ?>" />
+				<span><?php esc_html_e( 'Meetup', 'the-events-calendar' ); ?></span>
+			</td>
+			<td class="indicator <?php esc_attr_e( $indicator ); ?>"><span class="dashicons dashicons-<?php echo esc_attr( $indicator_icons[ $indicator ] ); ?>"></span></td>
+			<td><?php echo esc_html( $text ); ?></td>
+			<td><?php echo $notes; ?></td>
+		</tr>
+	</tbody>
+</table>
+
+
+<table class="event-aggregator-status">
+	<thead>
+		<tr class="table-heading">
+			<th colspan="4"><?php esc_html_e( 'Import Scheduler Status', 'the-events-calendar' ); ?></th>
+		</tr>
+	</thead>
+	<tbody>
+		<?php
+		$indicator = 'good';
+		$notes = '&nbsp;';
+
+		// @todo add API request for pingback check
+		if ( defined( 'DISABLE_WP_CRON' ) && true === DISABLE_WP_CRON ) {
+			$text = __( 'WP Cron is enable', 'the-events-calendar' );
```

Comment:
If `DISABLED_WP_CRON` is defined and is true, doesn't that mean it is _disabled_ rather than _enabled?_